Cargo.toml

```
@@ -28,9 +28,9 @@ arrow-schema = { git = "https://github.com/Coralogix/arrow-rs.git", tag="v48.0.0
 configure_me = { version = "0.4.0" }
 configure_me_codegen = { version = "0.4.4" }
 
-datafusion = { git = "https://github.com/apache/arrow-datafusion.git", rev = "af2cda928140f85d369e26931756a0cbc127e9ec", features=["avro"] }
-datafusion-cli = { git = "https://github.com/apache/arrow-datafusion.git", rev = "af2cda928140f85d369e26931756a0cbc127e9ec" }
-datafusion-proto = { git = "https://github.com/apache/arrow-datafusion.git", rev = "af2cda928140f85d369e26931756a0cbc127e9ec" }
+datafusion = { git = "https://github.com/Coralogix/arrow-datafusion.git", tag = "v32.0.0-cx.1", features=["avro"] }
+datafusion-cli = { git = "https://github.com/Coralogix/arrow-datafusion.git", tag = "v32.0.0-cx.1" }
+datafusion-proto = { git = "https://github.com/Coralogix/arrow-datafusion.git", tag = "v32.0.0-cx.1" }
```

Coralogix -> coralogix maybe?
```suggestion
datafusion = { git = "https://github.com/coralogix/arrow-datafusion.git", tag = "v32.0.0-cx.1", features=["avro"] }
datafusion-cli = { git = "https://github.com/coralogix/arrow-datafusion.git", tag = "v32.0.0-cx.1" }
datafusion-proto = { git = "https://github.com/coralogix/arrow-datafusion.git", tag = "v32.0.0-cx.1" }
```